clang/tools/dpct/DpctOptRules/cmake_script_migration_rule.yaml

```diff
@@ -3175,3 +3175,42 @@
       MatchMode: Full
       In: \${CMAKE_CUDA_RUNTIME_LIBRARY_LINK_OPTIONS_SHARED}
       Out: ""
+
+- Rule: rule_unsupported_flags_prec_div
```

Comment:
Please remove this,  as  it is already in [make_script_migration_rule_optional.yaml].

Reply:
I removed these rules but they appeared again while rebasing.
Removed now.